cmake/FindGENIE.cmake

```diff
@@ -0,0 +1,107 @@
+#
+# FindGENIE.cmake
+#
+# Find an installation of GENIE. Highly tuned to the installation
+# of GENIE that is built into the ldmx/dev image.
+#
+# This will define the following variables
+#
+# GENIE_FOUND
+# GENIE_INCLUDE_DIRS - path to the headers
+# GENIE_LIBRARIES    - CMake list of CMake targets
+#
+# and the following main target to link to
+# (other targets are defined for specific GENIE libraries)
+#
+# GENIE::GENIE
+#
+
+if (GENIE_FOUND)
+  return()
+endif()
+
+set(GENIE_INCLUDE_DIRS "/usr/local/include/GENIE")
```

Comment:
Do we want to update this to not include the `GENIE` subdirectory? This would then force all tine `#include` statements to have `GENIE` in them when using `GENIE` stuff (which might be helpful?)